src/internal/minikubetestenv/deploy.go

```diff
@@ -78,6 +131,14 @@ func UpgradeRelease(t *testing.T, ctx context.Context, kubeClient *kube.Clientse
 	return testutil.AuthenticatedPachClient(t, testutil.NewPachClient(t), user)
 }
 
+func InstallReleaseEnterprise(t *testing.T, ctx context.Context, kubeClient *kube.Clientset, user string) *client.APIClient {
```

Comment:
`*testing.T` -> `testing.TB`

TB is the common interface between tests and benchmarks, it makes this function slightly more useful although we don't have a ton of benchmarks.  Both work with the `require` package, and both support `Cleanup`.